RUNTIME ERROR: read testdata: is a directory
-------------------------------------------------
	During evaluation	

-------------------------------------------------
	testdata/import_failure_directory:3:1-10	$

import '.'


